tests/test_toggle.py

```diff
@@ -336,6 +358,38 @@ def test_dynamic_to_json(self):
         self.assertEqual(json.loads(json.dumps(data)), golden)
 
 
+class ToggleLoadTC(unittest.TestCase):
+
+    def test_load(self):
+        fixture = '''[{"fixed": {"show_axis": false}},
```

Comment:
@buganini, this is for your reference.

It's far from complete, but a starting point for how a json configuration file may look like.